### DISCUSSION TITLE
Is using `vite-node` recommend without vitest?

### DISCUSSION THREAD

**Discussion by togami2864** [vitest-dev/vitest#1209] [Q&A] [answered]
Hi👋🏼  Thank you for developing and maintaining great libraries.

Of course, I often use vitest as test-runner  and I also use `vite-node` a lot when coding in TypeScript. It provides me to a great developer experience.

I want everyone to know more about `vite-node`.
Is `vite-node` only the engine of vitest and not intended to be widely used on its own? `(e.g., aim to an alternative to nodemon)`

Thanks.

**Comment by sheremet-va** [comment, marked as answer]
vite-node was developed before vitest, and was not tight to it before 🤔 
I think it can safely be used outside of Vitest world :P
